samples/internat/internat.cpp

```diff
@@ -533,17 +531,8 @@ void MyFrame::OnPlay(wxCommandEvent& WXUNUSED(event))
     }
     else
     {
-        // this is a more implicit way to write _() but note that if you use it
-        // you must ensure that the strings get extracted in the message
-        // catalog as by default xgettext won't do it; it only knows of _(),
-        // not of wxTRANSLATE(). As internat's readme.txt says you should thus
-        // call xgettext with -kwxTRANSLATE.
-        str = wxGetTranslation(wxTRANSLATE("Bad luck! try again..."));
-
-        // note also that if we want 'str' to contain a localized string
-        // we need to use wxGetTranslation explicitly as wxTRANSLATE just
-        // tells xgettext to extract the string but has no effect on the
-        // runtime of the program!
+        // this is a more verbose way to write _()
+        str = wxGetTranslation("Bad luck! try again...");
```

Comment:
IMO, this change may not be for the best. 

Firstly, I think that all the strings for translation (except where `wxTRANSLATE()` is really needed) should be marked just with `_()` macro, i.e., without the need to add a `wxGetTranslation` to gettext keywords. I also don't see any advantage of using the function instead of the macro, unless one is using additional parameters of the function (not demonstrated in the sample?).

Reply:
The original idea was that you don't have to use `_()` if you don't want to/don't like it, so we do want to use `wxGetTranslation()` here.

OTOH I'd keep the part of the comment saying that you need to use `xgettext -kwxGetTranslation`.

Reply:
This is weird, I was absolutely certain that `_` was in the default keywords list, but according [to the docs](https://www.gnu.org/software/gettext/manual/html_node/xgettext-Invocation.html) it isn't. I wonder if this could have changed...

In any case, I think we should document `-kwxPLURAL` too.